web/client/utils/PluginsUtils.js

```diff
@@ -381,8 +386,39 @@ const PluginsUtils = {
     connect: (mapStateToProps, mapDispatchToProps, mergeProps, options) => {
         return connect(mapStateToProps, mapDispatchToProps, mergeProps || pluginsMergeProps, options);
     },
+    /**
+     * Use this function to export a plugin from a module.
+     *
+     * @param {string} name name of the plugin (without the Plugin postfix)
+     * @param {object} config configuration object, with the following (optional) properties:
+     *  * component: ReactJS component that implements the plugin functionalities, can be null if the plugin supports lazy loading
```

Comment:
I suggest to use 
```
@param {type} cfg.name for all the cfg object parameters, so it will can specify type and if they are options (using [])
```
also an 
```
@example
createPlugin( MyPlugin, {
   component: ...
   options: { 
      disablePluginIf: ,,,
   },
   containers: {
         Toolbar: {}
   }
}
```
could be useful. I see there are some usage samples around, but they are not so many neither complete.